### DISCUSSION TITLE
Is there a way to 'zero out' account views in the dashboard month to month?

### DISCUSSION THREAD

**Discussion by isaacgr** [firefly-iii/firefly-iii#6963] [Firefly III] [answered]
So in my dashboard right now I'm viewing my chequing, savings and investments. 

For chequing month to month I want to see whatever the current balance is and see how that moves with expenses.

However for savings and investments, I want to only see the balance for that month. Since these can get significantly higher than chequing (i.e. 10k in savings vs. 1k in chequing) the graph scale can get too high. 

So at the beginning of the month these views should reset to 0 and I can monitor them in the dashboard as they move for that month, but when viewing the actual account info I can see the full balance from the beginning of time.

I know this sounds specific, but not sure if that exists. Its really only something I'd like because the graph shows all accounts in the dashboard. So maybe an alternative would be to show something in the dashboard but not on the graph?

**Comment by JC5** [comment, marked as answer]
Sorry, there's not a possibility to do this.

**Reply by isaacgr** [reply]
Would you consider adding this as a feature if I put in a ticket?

**Reply by JC5** [reply]
Feel free, but given the amount of work this will not be picked up any time soon I'm afraid.

**Reply by isaacgr** [reply]
No problem. The same question yields the same response at my work.
